.all-contributorsrc

```diff
@@ -36,6 +36,315 @@
+      "avatar_url": "https://avatars.githubusercontent.com/u/82889095?v=4",
+      "profile": "https://github.com/AaronjSugarman",
+      "contributions": [
+        "code"
+      ]
+    },
+    {
+      "login": "karnauskas",
+      "name": "Marius Karnauskas",
+      "avatar_url": "https://avatars.githubusercontent.com/u/1094012?v=4",
+      "profile": "https://github.com/karnauskas",
+      "contributions": [
+        "code"
+      ]
+    },
+    {
+      "login": "GatienH",
+      "name": "GatienH",
+      "avatar_url": "https://avatars.githubusercontent.com/u/6501059?v=4",
+      "profile": "https://github.com/GatienH",
+      "contributions": [
+        "code"
+      ]
+    },
+    {
+      "login": "mvidonne",
+      "name": "Marie-Pierre Vidonne",
+      "avatar_url": "https://avatars.githubusercontent.com/u/46820105?v=4",
+      "profile": "https://github.com/mvidonne",
+      "contributions": [
+        "data",
+        "review"
+      ]
+    },
+    {
+      "login": "Amustache",
+      "name": "Stache",
+      "avatar_url": "https://avatars.githubusercontent.com/u/5108539?v=4",
+      "profile": "https://github.com/Amustache",
+      "contributions": [
+        "review",
+        "infra"
+      ]
+    },
+    {
+      "login": "afisher3578",
+      "name": "Alex Fisher",
+      "avatar_url": "https://avatars.githubusercontent.com/u/92438650?v=4",
+      "profile": "https://github.com/afisher3578",
+      "contributions": [
+        "code",
+        "infra"
+      ]
+    },
+    {
+      "login": "pdehaye",
+      "name": "Paul-Olivier Dehaye",
+      "avatar_url": "https://avatars.githubusercontent.com/u/3274335?v=4",
+      "profile": "https://github.com/pdehaye",
+      "contributions": [
+        "code"
+      ]
+    },
+    {
+      "login": "guillett",
+      "name": "Thomas Guillet",
+      "avatar_url": "https://avatars.githubusercontent.com/u/1410356?v=4",
+      "profile": "https://github.com/guillett",
+      "contributions": [
+        "code"
+      ]
+    },
+    {
+      "login": "Kissaki",
+      "name": "Jan Klass",
+      "avatar_url": "https://avatars.githubusercontent.com/u/93181?v=4",
+      "profile": "https://github.com/Kissaki",
+      "contributions": [
+        "code",
+        "review"
+      ]
+    },
+    {
+      "login": "Harshil-Jani",
+      "name": "Harshil Jani",
+      "avatar_url": "https://avatars.githubusercontent.com/u/79367883?v=4",
+      "profile": "https://github.com/Harshil-Jani",
+      "contributions": [
+        "code"
+      ]
+    },
+    {
+      "login": "AdrienFines",
+      "name": "Adrien Fines",
+      "avatar_url": "https://avatars.githubusercontent.com/u/41912392?v=4",
+      "profile": "https://github.com/AdrienFines",
+      "contributions": [
+        "code"
+      ]
+    },
+    {
+      "name": "Marine Guillaume",
+      "profile": "https://twitter.com/marineguillaum",
+      "avatar_url": "https://opentermsarchive.org/images/contributors/marine-guillaume.jpg",
```

Comment:
> In order to avoid the need for a CORS declaration, couldn't this be sufficient? 🙂 

```suggestion
      "avatar_url": "/images/contributors/marine-guillaume.jpg",
```

Reply:
No, the images are also displayed from the readme. Another path like `/public/images/contributors/...` could make the images displayed in the readme, but not on the website.